### PR TITLE
Automatically release version

### LINKED ISSUE #11: Release on master, not on tag
Change the GHA so that it creates a new release on all pushes to master.

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes #11

I've tested this on a fork and it works nicely. 

p.s. I'm updating the minor version and not the major one, which makes slightly more sense to me

### CODE REVIEW

File: .github/bin/set-credentials [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+git config --global user.email "github-actions[bot]@users.noreply.github.com"
+git config --global user.name "github-actions[bot]"
```
Comment (iHiD) [inline .github/bin/set-credentials:4]: Should we use the exercism bot instead?

**Comment by iHiD** [comment]
(cc @sleeplessbyte and @joshgoebel for visibility as you've both input on this)

**Comment by SleeplessByte** [comment]
Cool. Looks good.

Where is the version stored now? Just the release tag? Or did I overlook something.

**Comment by ErikSchierboom** [comment]
> Where is the version stored now? Just the release tag? Or did I overlook something.

@SleeplessByte It is stored in [tooling_webserver.nimble](https://github.com/exercism/tooling-webserver/blob/master/tooling_webserver.nimble#L3)

File: .github/bin/set-credentials (hunk 1 above)
Comment (ErikSchierboom) [reply .github/bin/set-credentials:4]: The exercism bot? Do we have one?

The above code is the one we are currently using in the v3 workflows.

**Comment by ErikSchierboom** [comment]
I'm gonna merge this to fix the C# and Ruby tooling Dockerfiles :) We can fix things later.

**Pull request merged**